upstream-community-operators/crossplane/crossplane.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
+- currentCSV: crossplane.v0.9.0
```

Comment:
I'm ignoring the [`master` channel](https://charts.crossplane.io/master) which is updated on each commit. While we could have CI/CD create a PR against this repo to keep the value maintained, it seems like that would create too much maintainer homework on the community-operators project.

The [alpha channel](https://charts.crossplane.io/alpha) has a more sensible throughput for maintaining bundles of this package.